README.md

```diff
@@ -53,6 +53,10 @@ Let's say a user raised an issue with the title `I can't get it to work on my co
 ```
 Derek set title: Question - does this work on Windows 10?
 ```
+or
+```
+Derek edit title: Question - does this work on Windows 10?
```

Comment:
thanks I keep typing this in, good change.

Reply:
The other pro-point for Derek is that it works fine on the mobile version of GitHub where as even if you have write access you can't take these actions.